bundles/org.openhab.binding.bticinosmarther/README.md

```diff
@@ -0,0 +1,328 @@
+# BTicino Smarther Binding
+
+The Smarther binding implements a bridge to the Legrand/BTicino Smarther v2.0 API and allows you to control your BTicino Smarther chronothermostat units with openHAB, making it possible to discover Smarther units connected to your Legrand/BTicino Smarther account.
+
+Smarther chronothermostat is produced by BTicino (https://www.bticino.com/products-catalogue/smarther-the-connected-thermostat/), has its own API set and doesn't support the OpenWebNet protocol.
+
+Companion HABPanel widget can be found here: [BTicino Smarther HABPanel widget](https://github.com/MrRonfo/openhab-smarther-widget)
```

Comment:
There should not be specific references to a UI in this README, especially not from external sources.
```suggestion
```